libcnb-test/tests/integration_test.rs

```diff
@@ -7,8 +7,128 @@
 // https://rust-lang.github.io/rust-clippy/stable/index.html
 #![warn(clippy::pedantic)]
 
-use libcnb_test::{BuildpackReference, TestConfig, TestRunner};
-use tempfile::tempdir;
+use indoc::indoc;
+use libcnb_test::{
+    assert_contains, assert_not_contains, BuildpackReference, PackResult, TestConfig, TestRunner,
+};
+use std::path::PathBuf;
+use std::time::Duration;
+use std::{env, fs, thread};
+
+#[test]
+#[ignore]
+fn basic_build() {
+    TestRunner::default().run_test(
+        TestConfig::new("heroku/builder:22", "test-fixtures/procfile").buildpacks(vec![
+            BuildpackReference::Other(String::from("heroku/procfile")),
+        ]),
+        |context| {
+            assert_eq!(context.pack_stderr, "");
```

Comment:
(Observation, follow up work?)

I like the following more here as it is more explicit what we're testing for:
```rust
assert!(context.pack_stderr.is_empty());
```

However, this would not log the contents of `context.pack_stderr` when it isn't empty, complicating debugging. Do we want an `assert_empty` macro in libcnb?